src/main/java/com/puppycrawl/tools/checkstyle/checks/design/OneTopLevelClassCheck.java

```diff
@@ -27,8 +27,8 @@
 
 /**
  * <p>
- * Checks that each top-level class, interface
- * or enum resides in a source file of its own.
+ * Checks that each top-level class, interface, enum
+ * or annotation resides in a source file of its own.
```

Comment:
same should be added in line 35 as well.

Reply:
Fixed